specifyweb/frontend/js_src/lib/components/Toolbar/WbsDialog.tsx

```diff
@@ -241,7 +241,7 @@ export function DataSetsDialog({
                           : undefined
                       }
                     >
-                      <TableIcon label={false} name="Workbench" />
+                      <TableIcon label name={dataset.uploadplan?.baseTableName ?? 'Workbench'} />
```

Comment:
FYI: `label` is equivalent to `label={true}`
before it was `label={false}`

after this change, the table name would be announced by screen reader (and have a tooltip) - before it wasn't because it would have been "Workbench" for all, thus useless

sounds good to me, just mentioning this to make sure this was intentional change

Reply:
That was my thought as I realized it would be something screen reader users might want to hear/see.